### ISSUE TITLE
Uncaught DOMException: Failed to construct 'PresentationRequest': The document is sandboxed and lacks the 'allow-presentation' flag

### ISSUE THREAD

**Issue by maxweb4u** [troybetz/react-youtube#384]
I have this mistake when installed latest version: 10.1.0

![error](https://user-images.githubusercontent.com/11961928/203413426-fd76aa4d-3234-450b-804e-9ce5f0173be7.png)

react-youtube: v10.1.0
react: 17.0.2

**Comment by Ojay** [comment]
Experiencing this too, same setup pretty much. I found [this](https://github.com/nasa8x/v-video-embed/issues/3) issue/fix on a different package which may help get this resolved.

**Comment by maxweb4u** [comment]
Yes

> Experiencing this too, same setup pretty much. I found [this](https://github.com/nasa8x/v-video-embed/issues/3) issue/fix on a different package which may help get this resolved.

yes I found this fix too, but it's on another package. And react-youtube doesn't provide an ability to add 'sandbox' property. So how could I add 'allow-presentation' to the sandbox for react-youtube.

**Comment by Ojay** [comment]
It was more posted for the devs to reference if/when they get to fixing it for this package. Unfortunately I don't think its possible to apply `allow-presentation` to `react-youtube` as is... 😕

**Comment by ruisaraiva19** [comment]
The YouTube iframe API already adds the `sandbox` attribute. Not sure we we should mess with it 👀

**Comment by yshterev** [comment]
`sandbox` seems to add `allow-same-origin allow-scripts allow-forms allow-popups allow-popups-to-escape-sandbox`
The error is about `allow-presentation` which is missing.

**Comment by rnrajurkar** [comment]
adding 
player.h.attributes.sandbox.value = "allow-same-origin allow-scripts allow-forms allow-popups allow-popups-to-escape-sandbox allow-storage-access-by-user-activation allow-presentation"; adds allow-presentation but does not solve the problem 
![image](https://user-images.githubusercontent.com/110887105/206156646-34d505c9-30f2-436f-bd5d-e7b3b6c64c7c.png)

**Comment by oknaus-huemor** [comment]
Yes, iframe has allow-presentation but error still appears in console

**Comment by RexHung0302** [comment]
Does anyone fixed this yet ? I got same error...

**Comment by joshuagraber** [comment]
Having the same issue. The `Options` type coming from `dist/youtube-player` appears to be incomplete. Would it be possible to extend this type and add support for adding our own `sandbox` attributes? If contributions are welcome, I would be happy to raise a PR for this. It doesn't affect functionality, but all that red in my console bothers me.
